pipes/run-pipe_local.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# Wrappers around Snakemake for use on the Broad UGER cluster
```

Comment:
Can we move this to `pipes/Broad_UGER/` instead of `pipes/`? Or is the comment here on line 2 incorrect--is this actually for running on a single VM, not on UGER?

Reply:
Sorry about that; the comment was incorrect. It is indeed for invoking Snakemake on a single instance with similar params. I'll fix the comment.